roles/openshift_hosted_logging/tasks/deploy_logging.yaml

```diff
@@ -28,21 +28,20 @@
     register: secret_output
     failed_when: "secret_output.rc == 1 and 'exists' not in secret_output.stderr"
 
-  - name: "Copy serviceAccount file"
-    copy:
-      dest: /tmp/logging-deployer-sa.yaml
-      src: "{{role_path}}/files/logging-deployer-sa.yaml"
-      force: yes
-
-  - name: "Create logging-deployer service account"
+  - name: "Create templates for logging accounts and the deployer"
     command: >
-      {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f  /tmp/logging-deployer-sa.yaml
-    register: deployer_output
-    failed_when: "deployer_output.rc == 1 and 'exists' not in deployer_output.stderr"
+      {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f /usr/share/openshift/examples/infrastructure-templates/enterprise/logging-deployer.yaml
+    register: template_output
+    failed_when: "template_output.rc == 1 and 'exists' not in template_output.stderr"
+
+  - name: "Process the logging accounts template"
+    shell:  "{{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig process logging-deployer-account-template |  {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f -"
```

Comment:
> for me it fails here because logging-deployer-account-template is in openshift namespace

Are you redeploying on top of an earlier deployment?  Because doing a fresh deployment does this:
```
  - name: "Create logging project"
    command: >
      {{ openshift.common.admin_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig new-project logging
    when: logging_project_result.stdout == ""

  - name: "Changing projects"
    command: >
      {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig project logging
...
  - name: "Create templates for logging accounts and the deployer"
    command: >
      {{ openshift.common.client_binary }} --config={{ mktemp.stdout }}/admin.kubeconfig create -f /usr/share/openshift/examples/infrastructure-templates/enterprise/logging-deployer.yaml
    register: template_output
    failed_when: "template_output.rc == 1 and 'exists' not in template_output.stderr"
```
AFAICT, this will create the `logging-deployer-account-template` in the `logging` namespace.